server/src/services/apiClient.ts

```diff
@@ -0,0 +1,28 @@
+import fetch, { Headers } from 'cross-fetch'
+
+import { env } from '../libs/environment/env'
+
+import { ENTITY_MAP, EntityKeys } from './entities/types'
+
+const { API_BASE_URL, API_BASE_PATH_NATIVE_V1 } = env
+
+const { href } = new URL(API_BASE_URL)
+
+export async function apiClient(type: EntityKeys, id: number) {
+  const entityValue = ENTITY_MAP[type]
+  if (!entityValue) {
+    throw new Error(`Unknown entity: ${type}`)
+  }
+
+  const url = `${href}${API_BASE_PATH_NATIVE_V1}/${entityValue.NAME}/${id}`
+
+  const response = await fetch(url, {
+    headers: new Headers({
+      'cache-control': 'public,max-age=3600',
+    }),
+  })
+  if (response.ok && response.status === 200) {
```

Comment:
c'est redondant la propriété `ok` vérifie simplement si le code de retour est entre [200 et 299]

https://fetch.spec.whatwg.org/#ok-status

```suggestion
  if (response.ok) {
```

ou si le code ne pas pas etre un autre 2XX (pourquoi pas)

```suggestion
  if (response.status === 200) {
```

Reply:
Agree